src/main/java/org/spongepowered/api/text/dialogue/Question.java

```diff
@@ -0,0 +1,205 @@
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.api.text.dialogue;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import org.spongepowered.api.Sponge;
+import org.spongepowered.api.text.Text;
+import org.spongepowered.api.text.TextRepresentable;
+import org.spongepowered.api.text.channel.MessageChannel;
+import org.spongepowered.api.util.ResettableBuilder;
+import org.spongepowered.api.util.Tristate;
+
+import java.util.Collection;
+
+/**
+ * Each {@link DialogueArchetype} is made up of a dynamic tree of {@link Question}s,
+ * meaning that each {@link Question} chooses the next one. A {@link Question}
+ * is responded to by a {@link Speaker} with an {@link Answer}.
+ */
+public interface Question extends TextRepresentable {
+
+    /**
+     * Creates a {@link Builder} to return an {@link Question}.
+     *
+     * @return The new builder
+     */
+    static Builder builder() {
+        return Sponge.getRegistry().createBuilder(Builder.class);
+    }
+
+    /**
+     * Gets the {@link Text} that this question shows when asked.
+     *
+     * @return The text object
+     */
+    Text getText();
+
+    @Override
+    default Text toText() {
+        return getText();
+    }
+
+    /**
+     * Gets the {@link AnswerProcessor} to use when processing {@link Answer}s.
+     *
+     * @return The processor
+     */
+    AnswerProcessor getMainProcessor();
+
+    /**
+     * Gets the {@link AdditionalAnswerProcessor}s to use when processing
+     * {@link Answer}s. This {@link Collection} is immutable.
+     *
+     * @return The processor
+     */
+    Collection<AdditionalAnswerProcessor> getAdditionalProcessors();
+
+    /**
+     * Gets all {@link MessageChannel}s that are allowed through suppression.
+     * This {@link Collection} is immutable.
+     *
+     * @return The channels
+     */
+    Collection<MessageChannel> getAllowedChannels();
+
+    /**
+     * Gets whether or not this question suppresses messages.
+     *
+     * @return Whether or not messages are suppressed.
+     */
+    Tristate suppressesMessages();
```

Comment:
@ST-DDT Questions are not necessarily aware of their Dialogue. Tristate is used to allow for the DialogueArchetype to define one; note that the identical methods in DialogueArchetype are primitive boolean. It was either Tristate or @Nullable Boolean.